input/kafkamdm/kafkamdm.go

```diff
@@ -349,15 +351,30 @@ func (k *KafkaMdm) consumePartition(topic string, partition int32, currentOffset
 }
 
 func (k *KafkaMdm) handleMsg(data []byte, partition int32) {
-	md := schema.MetricData{}
-	_, err := md.UnmarshalMsg(data)
+	pointMsg := msg.Point{}
+	if len(data) == 29 && data[0] == 0 {
```

Comment:
I think that the unmarshaling of bytes to a msg.Point should be handled in the `msg` package.

Reply:
moved it